tests/imageview/test_imageview.py

```diff
@@ -12,6 +13,31 @@ def test_nan_image():
     app.processEvents()
     v.window().close()
 
+
+def test_timeslide_snap():
+    count = 31
+    frames = np.ones((count, 10, 10))
+    iv = pg.ImageView(discreteTimeLine=True)
+    assert iv.nframes() == 0
+    iv.setImage(frames, xvals=(np.linspace(0., 1., count)))
+    iv.show()
+    assert iv.nframes() == count
+    speed = count / 2
+    iv.play(speed)
+    assert iv.playRate == speed
+    iv.timeLine.setPos(0.51)
+    assert iv.playRate == 0
+    ind, val = iv.timeIndex(iv.timeLine)
+    assert ind == count // 2
+    assert val == 0.5
+    iv.togglePause()
```

Comment:
perhaps we should add a comment here indicating that this call to `togglePause` _should_ get playback going?